01-import_and_maxfilter.py

```diff
@@ -273,8 +273,11 @@ def run_maxwell_filter(subject, session=None):
         os.makedirs(os.path.dirname(raw_fname_out), exist_ok=True)
 
         raw_out.save(raw_fname_out, overwrite=True)
-        if config.plot:
-            raw_out.plot(n_channels=50, butterfly=True)
+        if config.interactive:
+            (raw_out
+             .copy()
```

Comment:
why do you copy? you don't want to let user set bad channels or set annotations?

Reply:
Very good point. I copy() because I want to only plot MEG channels, because only they are touched during Maxwell filtering. Apparently there is no `pick` parameter in `Raw.plot()`, therefore I copy'd before picking (see your next question). But you're right, this won't allow users to set bads manually. My idea was to later extend it such that we extract the bads from the copy's info and inject them into the original data that will be saved to disk… Yes, seems awkward…